### ISSUE TITLE
Screen not readable in normal daylight

### ISSUE THREAD

**Issue by nzwaneveld** [nzwaneveld/SmartHalo-User-Group#8]
Is there a way to change the intensity of the screen? I took the SmartHalo 2 for a test drive and found that it wasn't easy to read the street names on a sunny day. I was not able to change the intensity or contrast of the screen. 

** Tim Meskens asked the same question and got a reply from the SH team saying that the main OLED display is not that bright, so they make the halo extra bright during daylight hours and used a large font. Their only real tip was to tilt the unit a bit, which unfortunately doesn't really improve the readability.

Affected users:
Nico Zwaneveld
Tom Meskens
Roelof van Beek
Matthew Garcia
Rory Meijer
Vasily Shmelev
Waldo Jeffers
Christoph Lemmer
Marian Schmidt

**Comment by nzwaneveld** [comment]
SmartHalo - May 18, 2021: It’s true PeekDisplay can be harder to read in some conditions. Here are some tips and explanations for that: https://help.smarthalo.bike/hc/en-us/articles/1500009840882-PeekDisplay-is-not-bright-enough-when-it-s-sunny

**Note: THIS IS NOT AN ACCEPTABLE SOLUTION. Keep this on Pending**

**Comment by Mubaer** [comment]
I second that. The main display is next to unreadable in sunlight and light cloudy conditions. In bright daylight, even the LEDs are hard to read. This produce is a pile of unfinished stinking pile of garbage not worth using up my time.
Using a random app on an average smartphone is better, safer, more flexible.

**Comment by caetienne** [comment]
I totally agree, even in daylight with rain, it's really difficult de read it properly

**Comment by stephanepapin** [comment]
+1
